### ISSUE TITLE
Warning message in build: MSB3270: There was a mismatch between the processor architecture of the project being built 

### ISSUE THREAD

**Issue by ACueva** [Esri/distance-direction-addin-dotnet#509]
```
  C:\Program Files (x86)\MSBuild\12.0\bin\Microsoft.Common.CurrentVersion.targets(1635,5): warning MSB3270: There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "ESRI.ArcGIS.ADF.Connection.Local, Version=10.3.0.0, Culture=neutral, PublicKeyToken=8fc3cc631e44ad86, processorArchitecture=x86", "x86". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and references, or take a dependency on references with a processor architecture that matches the targeted processor architecture of your project. [c:\jenkins\workspace\Add-ins\DD\DD_Pipeline\distance-direction-addin-dotnet\source\addins\ArcMapAddinDistanceAndDirection\ArcMapAddinDistanceAndDirection\ArcMapAddinDistanceAndDirection.csproj]

    1 Warning(s)
    0 Error(s)
```

**Comment by csmoore** [comment]
Addressed in PR #511 - please retest after merge.

Technically this is an MSBuild warning vs. compiler warning - but we can fix

**Comment by topowright** [comment]
![image](https://user-images.githubusercontent.com/5488313/36756926-46084196-1bc5-11e8-8a5a-d4b6dddc8492.png)
This has been verified.
